### DISCUSSION TITLE
150+ deployments

### DISCUSSION THREAD

**Discussion by Aaditya-23** [vercel/community#3592] [Help] [answered]
### Summary

Now how do I deal with this? It does not cause any problems but its annoying. Deleting old deployments from vercel does not fix it.

![Screenshot 2023-08-13 191204](https://github.com/vercel/community/assets/74983108/6b9d5af6-7b34-4c2a-98c0-7a4cc11c9406)

### Steps to Reproduce

Every push to git creates another deployment.

**Comment by amyegan** [comment, marked as answer]
Sorry @Aaditya-23. If you would like the disable automatic deployments, that can be done by configuring [`git.deploymentEnabled`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled) in a `vercel.json` file. Or you can get more control over which things to automatically deploy by configuring [Ignored Build Step](https://vercel.com/docs/concepts/projects/overview#ignored-build-step).

If you're asking about something else, please let me know 🙏
